### PR TITLE
Feat/Count queries

### PR DESCRIPTION
**Pull request by danstarns**
# Description

Adds top-level queries to count nodes. 

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [x] Documentation has been updated
- [x] TCK tests have been updated
- [x] Integration tests have been updated
- [x] Example applications have been updated
- [x] New files have copyright header
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Review by darrellwarde — APPROVED** [review]
This looks great, and will be very well received by many! Just one comment about naming really!

File: packages/graphql/src/schema/make-augmented-schema.ts [hunk 1]
```diff
@@ -1148,6 +1155,10 @@ function makeAugmentedSchema(
             composer.Query.addFields({
                 [pluralize(camelCase(node.name))]: findResolver({ node }),
             });
+
+            composer.Query.addFields({
+                [`count${pluralize(node.name)}`]: countResolver({ node }),
```
[the remark this answers is not in the document]
Comment (danstarns) [reply packages/graphql/src/schema/make-augmented-schema.ts:1160]: I thought about this for a while, I settled on `countMovies` to be consistent with `createMovies`, `deleteMovies`, and `updateMovies`.

**Comment by danstarns** [comment]
> This looks great, and will be very well received by many! Just one comment about naming really!

Thanks, I just realized. I forgot about the OGM so will add a method onto that sometime soon and bump this when it's ready again.

File: packages/graphql/src/schema/make-augmented-schema.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (darrellwarde) [reply packages/graphql/src/schema/make-augmented-schema.ts:1160]: Oh yeah, I think we discussed this before! However, just as a counter to that point, when executing a Query you're asking for what you want back, whilst when executing a Mutation you're specifying the action you want to happen.

So the Mutation naming makes sense, you're asking to "create movies" or "update movies", etc.

For the current queries, you ask for "movies" and get movies back... But with this you're asking for "count movies" and getting an integer back. The naming would make more sense if you were asking for a "movies count" based on a predicate.

**Comment by danstarns** [comment]
@darrellwarde I've made the changes we discussed and added a method for the OGM. Ill hold of merging just so you can have a last look.

**Review by darrellwarde — APPROVED** [review]
Excellent, looks good to me! 😄

**Pull request merged**